build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val higherKindCore = project settings (
 lazy val core = project dependsOn (opticsCore, higherKindCore) settings (
   defaultSettings,
   publishName := "core",
-  libraryDependencies ++= Seq(catsCore, catsEffect, catsTagless),
+  libraryDependencies ++= Seq(catsCore, catsEffect, catsTagless, derevo, magnolia),
```

Comment:
No, please don't add the `magnolia`and `derevo` dependencies to the `higherKindedCore`

Reply:
I suppose this type should be in the `core` or `data` modules, and derivation in the `data-derivation` module.